scripts/kind-e2e/e2e.sh

```diff
@@ -123,6 +125,23 @@ function setup_broker() {
     SUBMARINER_BROKER_TOKEN=$(kubectl --context=cluster1 -n ${SUBMARINER_BROKER_NS} get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='${SUBMARINER_BROKER_NS}-client')].data.token}"|base64 --decode)
 }
 
+function setup_broker_subctl() {
+    if kubectl --context=cluster1 get crd clusters.submariner.io > /dev/null 2>&1; then
+        echo Submariner CRDs already exist, skipping broker creation...
+    else
+        echo Installing broker on cluster1.
+        subctl --kubeconfig ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster1 deploy-broker --no-dataplane
```

Comment:
Will it work with the fix of #75 (Flip the dataplane subctl flag) ?

Reply:
We will update that flag when we update to subctl 0.0.3 :-), 0.0.2 should remain the same